code/datums/emergency_calls/zombie.dm

```diff
@@ -0,0 +1,43 @@
+
+/datum/emergency_call/zombie
```

Comment:
Same here.

Reply:
this happened apparently because this file was deleted in a pr after i branched.